### PR TITLE
Measuring impact of QuickLink in sites guide

### PR DESCRIPTION
**Pull request by demianrenzulli**
New guide for Quicklink site: "Measuring impact of QuickLink in sites".

@addyosmani: A new section "Measure" has been added to the nav bar. As a result  menu option appear closer to each other. Perhaps you have some better ideas on how to better accommodate this?
Another option would be to move this guide as a link inside the "Approach" section instead of on its own.

### CODE REVIEW

File: site/src/measure.njk [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+---
+{% extends "layouts/normal-section-wrapper.njk" %}
+{% block section %}
+{% markdownConvert %}
+## Measuring impact of QuickLink in sites
+
+Implementing Quicklink in sites can speed up navigations, by automatically prefetching in-viewport links during idle time.
+Different metrics can be improved as a result of this, the most common ones being [Start Render](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/quick-start-quide#TOC-Start-Render:) and [First Contentful Paint](https://developers.google.com/web/tools/lighthouse/audits/first-contentful-paint).
+
+In this section, we explore different ways of measuring the impact of Quicklink in sites. To showcase that, we’ll use [this demo](https://mini-ecomm.glitch.me/), consisting of a listing and product page. We'll compare performance of that site with [this other version](https://mini-ecomm-quicklink.glitch.me/) that requests and initializes the library in the product listing page. You can check the head of the page, to see the following code before the closing tag:
```
Comment (addyosmani) [inline site/src/measure.njk]: Could we expand `this demo` to link a link to the demo as well as a source link on Glitch for folks that want to fork it or view source?

File: site/src/measure.njk [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+```js
+<script src="https://unpkg.com/quicklink@1.0.1/dist/quicklink.umd.js"></script>
```
Comment (addyosmani) [inline site/src/measure.njk]: Would you like to use `https://unpkg.com/quicklink@latest/dist/quicklink.umd.js` here as it's a guide and should be okay to point to the latest version?

File: site/src/measure.njk [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+{% endmarkdownConvert %}
+
+<img class="article-image" src="/assets/images/screenshots/wpt-video-comparison.gif" height="435px" width="600px">
+
+{% markdownConvert %}
+
+### Using RUM (Real user monitoring) tools
+
+RUM tools, let you visualize how different metrics evolve in time for real users. If prefetching affects a large amount of pages, you might be able to see more page loads being loaded faster after implementing it, which can be reflected in metrics First Contentful Paint.
```
Comment (addyosmani) [inline site/src/measure.njk]: This is currently a long paragraph... :) Could we split this into 2-3 paragraphs to break it up just a little for readability?

File: site/src/measure.njk [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+For example, the [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report/) provides user experience metrics for how real-world Chrome users experience popular destinations on the web.
+CrUX data is available in [PageSpeedInsights](https://developers.google.com/speed/pagespeed/insights/) and also in [BigQuery](https://bigquery.cloud.google.com/dataset/chrome-ux-report:all?pli=1), but you can obtain a quick visualization of the evolution of your metrics using the CrUX dashboard [here](https://g.co/chromeuxdash). [This guide](https://web.dev/chrome-ux-report-data-studio-dashboard/) explains the process in detail.
```
Comment (addyosmani) [inline site/src/measure.njk]: Can we change this to `PageSpeed Insights`?

File: site/src/measure.njk [hunk 5]
```diff
@@ -0,0 +1,165 @@
+---
+title: Quicklink
+layout: layouts/base.njk
+description: Faster subsequent page-loads by prefetching in-viewport links during idle time.
+---
+{% extends "layouts/normal-section-wrapper.njk" %}
+{% block section %}
+{% markdownConvert %}
+## Measuring impact of QuickLink in sites
```
Comment (addyosmani) [inline site/src/measure.njk]: Quicklink :)

**Comment by demianrenzulli** [comment]
Hi @addyosmani, thanks for taking the time to review the article.
Just added your suggestions.

**Comment by addyosmani** [comment]
Re-opened the PR as I think it may have been accidentally closed... :)

**Comment by demianrenzulli** [comment]
Thanks @addyosmani. Let me know if you need anything else. 
Otherwise, feel free to merge.

**Comment by addyosmani** [comment]
Let's merge!

**Comment by demianrenzulli** [comment]
@addyosmani I've realized https://teejungle.net/ uses Quicklink! 😀

Wondering if, at some point we should re-write the measurement guide using that site, since it's a real production. site instead of just a demo?

**Pull request merged**